hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1923,6 +1923,18 @@ components:
           type: string
         service_endpoints:
           $ref: '#/components/schemas/ServiceEndpoints'
+        stake:
+          description: The node consensus weight at the end of the staking period
```

```suggestion
          description: The node consensus weight at the beginning of the staking period
```

same change to the description of `stake_rewarded` and `stake_total`

I understand the description is from the protobuf and our domain class, however the `on epochDay` is dropped so now they are invalid.

The associated staking_period in rest response is for `epochDay + 1`, so we need to change it from `at the end of` to `at the beginning of`

---

I've updated the openapi as indicated.